### DISCUSSION TITLE
TabBar in Xamarin Forms Shell displays text above icons (when referencing Xamarin.Google.Android.Material)

### DISCUSSION THREAD

**Discussion by jtorvald** [xamarin/Xamarin.Forms#15275] [Q&A]
Not sure if it belongs here but I post it anyway to maybe get an understanding.

I'm running the latest Xamarin Forms with Shell and noticed that on Android the Bottom Navigation Tabs rendered the text above the icon, instead of rendering them under the icon as it does in all the examples and iOS code.

I have no custom renderers for shell so at first I could not figure out what it was.
Then I saw: Xamarin.Google.Android.Material  Version="1.5.0.1" between my packages on Android.
I removed it and now the tabbar is back to normal.

So my questions are: 

- Why does this happen? 
- Do I need this package running Xamarin Forms on Android?

**Comment by trevoriancox** [comment]
We pinned this to 14.4.0.2 as in a slightly newer version, the text is moved to the top. And worse, the text overlaps the icon. We have this library only because another library is dependent on it.
